Stripe/STPBankAccount.m

```diff
@@ -24,6 +27,33 @@ @implementation STPBankAccount
 
 @synthesize routingNumber, country, currency, accountHolderName, accountHolderType;
 
+#pragma mark - STPBankAccountStatus
+
++ (NSDictionary<NSString *, NSNumber *> *)stringToStatusMapping {
+    return @{
+             @"new": @(STPBankAccountStatusNew),
```

Comment:
> I generally like this dictionary style of doing string <-> enum mappings and we use it a lot of places, but it does somewhat worry me as this method does not create a compiler error if we add a new enum value (whereas doing a switch statement without a default case does).

Yeah I agree there's room for improvement here but at least a test case will fail now if you incompletely add/remove an enum case.